src/__tests__/http-test.ts

```diff
@@ -13,8 +11,8 @@ import multer from 'multer';
 import { expect } from 'chai';
 import { describe, it } from 'mocha';
 import {
-  type ASTVisitor,
-  type ValidationContext,
+  ASTVisitor,
+  ValidationContext,
```

Comment:
Please extract into TS type imports?

Reply:
I'm not clear on what you mean. Do you want these as a separate block of imports?

Reply:
I mean `import type {` syntax supported by TS.

Reply:
Oh 🤦

Reply:
Also, can you do this as separate PR for all code in this repo
https://github.com/gajus/eslint-plugin-flowtype#eslint-plugin-flowtype-rules-type-import-style